openvino_node/config/face_detection.yaml

```diff
@@ -0,0 +1,6 @@
+device: CPU
+type: SSD
+model: /home/intel/workspace/rdk_ws/data/FP32/face-detection-adas-0001.xml
+weights: /home/intel/workspace/rdk_ws/data/FP32/face-detection-adas-0001.bin
+input_topic: /camera/color/image_raw
+output_topic: /openvino/detected_faces
```

Comment:
I am thinking of remapping these topics in launch files instead of declare them in yaml files in future. Keep as it is in this PR anyway.